server/game/game.js

```diff
@@ -653,6 +653,27 @@ class Game extends EventEmitter {
             }
         }
 
+        if(message.indexOf('/give-control') === 0) {
+            this.promptForSelect(player, {
+                activePromptTitle: 'Select a character',
+                waitingPromptTitle: 'Waiting for opponent to give control',
+                cardCondition: card => card.location === 'play area' && card.controller === player,
```

Comment:
I'll add discard + dead as valid targets. Hand + draw deck seem invalid, probably don't want take control to work w/ plots, agendas, or faction cards.